test/e2e/feature/feature.go

```diff
@@ -279,6 +279,10 @@ var (
 	// TODO: document the feature (owning SIG, when to use this feature for a test)
 	KubeProxyDaemonSetMigration = framework.WithFeature(framework.ValidFeatures.Add("KubeProxyDaemonSetMigration"))
 
+	// Owner: sig-network
+	// Marks tests that require the kernel to have support for the nfacct subsystem
```

Comment:
```suggestion
	// Marks tests that require the kernel to have support for the nfacct subsystem.
	// (Some distros don't include this in the kernel.)
```

(Just to make it clear that this isn't just a matter of setting a sysctl or something.)